Tests/ServiceTests/StringFilterTests.cs

```diff
@@ -0,0 +1,240 @@
+//using System;
```

Comment:
A complete class in commentary?

Reply:
I will remove the comments, the unit test is still valid and we might want to use the `Filter` object in the near future.